templates/appium.service.j2

```diff
@@ -7,6 +7,7 @@ After=network.target
 Type=simple
 User={{ appium_user }}
 ExecStart=/home/{{ appium_user }}/run-appium
```

Comment:
I think the `/run-appium` script is no longer needed, and we can just run `/usr/bin/appium` instead?